src/test/java/com/znsio/sample/e2e/entities/SAMPLE_TEST_CONTEXT.java

```diff
@@ -8,4 +8,6 @@
     public static final String MEETING_ID = "meetingId";
     public static final String MEETING_PASSWORD = "meetingPassword";
     public static final String INVITATION_LINK = "invitationLink";
+    public static final String USER_NAME = "raghav.gargril291900@gmail.com";
```

Comment:
Username should be passed from BL or step class
Context class and its variable are used for the Test level data crawling